.github/workflows/build.yml

```diff
@@ -128,7 +128,7 @@ jobs:
       - name: Wait sonar to process report
         uses: jakejarvis/wait-action@master
         with:
-          time: '60s'
+          time: '120s'
       - name: SonarQube Quality Gate check
```

Comment:
Probably instead of waiting 120s better to follow the strategy, we adopt for connect-cli.
It means a job just for Sonar that depends on previous builds. It seems that quality check fails only when repeated many times due to a test matrix.

Please take a look  https://github.com/cloudblue/connect-cli/blob/a02728c5c991ecf76768da6331bcfe76d6c863c4/.github/workflows/build.yml#L37

Reply:
I will leave it up to you :)))